{"name": "Упешная авторизация курьера при вводе валидных данных", "status": "passed", "start": 1739189245123, "stop": 1739189245306, "uuid": "2df30d63-f849-49ff-92f7-f9d98e5a059a", "historyId": "e8983cb6a968b5be42674fd046db981a", "testCaseId": "e8983cb6a968b5be42674fd046db981a", "fullName": "tests.test_login_courier.TestCourierLogin#test_courier_successful_login_valid_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "12AM"}, {"name": "thread", "value": "5788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}